### PR TITLE
feat(bulkmetadata): add DocumentViewer component

### PR DESCRIPTION
**Pull request by johan-hage**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
-		],
-		'react/prop-types': 0,
-	},
-	settings: {
-		react: {
-			version: 'detect', // Tells eslint-plugin-react to automatically detect the version of React to use
-		},
-	},
-};
+  env: {
```
Comment (fortunato) [inline .eslintrc.js:2]: Can you make sure you respect .editorconfig in your IDE? Or did the storybook ESLint install mess with this?
Can you open and re-save in that case?

File: package.json [hunk 2]
```diff
@@ -62,12 +62,12 @@
     "@babel/preset-react": "^7.13.13",
     "@babel/preset-typescript": "^7.13.0",
     "@babel/runtime": "^7.14.0",
-    "@storybook/addon-a11y": "^6.2.9",
-    "@storybook/addon-actions": "^6.2.9",
-    "@storybook/addon-docs": "^6.3.12",
-    "@storybook/addon-essentials": "^6.2.9",
-    "@storybook/addon-links": "^6.2.9",
-    "@storybook/react": "^6.2.9",
+    "@storybook/addon-a11y": "^6.4.8",
```
Comment (fortunato) [inline package.json:65]: Latest and greatest :muscle: :slightly_smiling_face:

File: src/components/DocumentViewer/DocumentViewer.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import DocumentViewer from './DocumentViewer';
+
+export default {
+	component: DocumentViewer,
+	title: 'DocumentViewer',
+} as Meta;
+
+export const PNGDocument = () => <DocumentViewer url="/assets/example.png" />;
```
Comment (fortunato) [inline src/components/DocumentViewer/DocumentViewer.stories.tsx:10]: Have you considered decorating these with a Type and a storyName? I.e.:
```
export const SingleFile: Story = () => (
	<FileUpload
		{...defaultProps}
		options={{ noClick: true, noKeyboard: true, maxFiles: 1 }}
		placeholder="Sleep een bestand in dit vlak of"
	/>
);
SingleFile.storyName = 'Single file';
```

File: src/components/DocumentViewer/DocumentViewerStyles.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import DocViewer from 'react-doc-viewer';
+
+export const DocumentViewerStyle = styled.div`
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+`;
+
+export const DocViewerStyle = styled(DocViewer)`
```
Comment (fortunato) [inline src/components/DocumentViewer/DocumentViewerStyles.ts:10]: Can you proxy the DocViewer props on this?
`styled(DocViewer)<Props>`

**Review by fortunato — CHANGES_REQUESTED** [review]
Nice work

File: src/components/DocumentViewer/DocumentViewerStyles.ts (hunk 4 above)
Comment (johan-hage) [reply src/components/DocumentViewer/DocumentViewerStyles.ts:10]: done

File: src/components/DocumentViewer/DocumentViewer.stories.tsx (hunk 3 above)
Comment (johan-hage) [reply src/components/DocumentViewer/DocumentViewer.stories.tsx:10]: Added type to story.

**Pull request closed without merging**